flink/v1.17/flink/src/test/java/org/apache/iceberg/flink/TestHelpers.java

```diff
@@ -483,7 +473,7 @@ private static void assertArrayValues(
 
   private static void assertMapValues(
       Types.MapType mapType, LogicalType type, Map<?, ?> expected, MapData actual) {
-    Assert.assertEquals("map size should be equal", expected.size(), actual.size());
+    assertThat(actual.size()).as("map size should be equal").isEqualTo(expected.size());
```

Comment:
this should use `hasSameSizeAs(...)`

Reply:
@nastra , I have problem using `hasSameSizeAs(...)` here as MapData is not extending from `Collection`, `assertThat` has difficulty inferring that `actual` can be enumerated.